exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+const numberOfTrainees = 15;
+const numberOfMentors = 8;
+const total = numberOfTrainees + numberOfMentors;
+const pOfTrainees = (numberOfTrainees / total) * 100;
+const pOfMentors = Math.round((numberOfMentors / total) * 100);
+
+
+console.log(`Percentage of trainees ${Math.round(pOfTrainees)}%`);
+console.log(`Percentage of mentors ${Math.round(pOfMentors)}%`);
```

Comment:
The value of `pOfMentors` was already rounded on line 5. Calling `Math.round` again here is unnecessary.

Reply:
Sure! I deleted it.